operations/power_management/Cray_Advanced_Platform_Monitoring_and_Control_CAPMC.md

```diff
@@ -19,15 +21,20 @@ for detailed information about API options and features.
 The `cray capmc` command (see `--help`) can be used to control power to
 specific components by specifying the component NID, component name (xname), or group.
 
-- [Power control and query by xname](#power-control-and-query-by-xname)
-  - [Controllable components](#controllable-components)
-  - [Power capping](#power-capping)
-  - [Power control and query by NID](#power-control-and-query-by-nid)
-  - [Power control and query by group](#power-control-and-query-by-group)
-- [Node energy](#node-energy)
-  - [System monitor](#system-monitor)
-  - [Others](#others)
-- [Deprecated interfaces](#deprecated-interfaces)
+- [Cray Advanced Platform Monitoring and Control (CAPMC)](#cray-advanced-platform-monitoring-and-control-capmc)
+    - [Power control and query by xname](#power-control-and-query-by-xname)
+        - [Controllable components](#controllable-components)
+            - [Air-cooled cabinets](#air-cooled-cabinets)
+            - [Liquid-cooled cabinets](#liquid-cooled-cabinets)
+            - [Naming convention](#naming-convention)
+                - [Examples of valid xnames](#examples-of-valid-xnames)
+        - [Power capping](#power-capping)
+        - [Power control and query by NID](#power-control-and-query-by-nid)
+        - [Power control and query by group](#power-control-and-query-by-group)
+    - [Node energy](#node-energy)
+        - [System monitor](#system-monitor)
+        - [Others](#others)
+    - [Deprecated interfaces](#deprecated-interfaces)
```

Comment:
```suggestion
- [Power control and query by xname](#power-control-and-query-by-xname)
    - [Controllable components](#controllable-components)
        - [Air-cooled cabinets](#air-cooled-cabinets)
        - [Liquid-cooled cabinets](#liquid-cooled-cabinets)
        - [Naming convention](#naming-convention)
            - [Examples of valid xnames](#examples-of-valid-xnames)
    - [Power capping](#power-capping)
    - [Power control and query by NID](#power-control-and-query-by-nid)
    - [Power control and query by group](#power-control-and-query-by-group)
- [Node energy](#node-energy)
    - [System monitor](#system-monitor)
    - [Others](#others)
- [Deprecated interfaces](#deprecated-interfaces)
```

Reply:
This can be handled with a different ticket.

Reply:
Fixed